docs/kb/developer-resources/running-commands-on-the-host-os.md

```diff
@@ -122,7 +122,7 @@ components:
               my_config_val='{{repl ConfigOption "my_config_option"}}'
 
               ## Complete example:
-              if [ "{{repl ConfigOption \"update_management_console\" }}" -eq "true" ]; then
+              if [ '{{repl ConfigOption "update_management_console" }}' -eq "true" ]; then
```

Comment:
i know you didnt write this but `-eq` is reserved for integer expressions. wanna fix the bug?